_data/committers.yml

```diff
@@ -176,6 +180,10 @@
   alias: yibocai
   affiliation: Arm
 
+- name: Alenka Frim
+  role: Committer
+  alias: alenka
```

Comment:
```suggestion
  alias: alenkaf
```
... if this is the JIRA profile alias, it should be alenkaf 😊

Reply:
Oh, in the phonebook there is `alenka` profile as committer? Oh, what a mess :D

Reply:
Created an INFRA issue, hope they can help =) For now, just leave it as is Rok, thanks!